shell/src/integration-test/kotlin/net/corda/tools/shell/InteractiveShellIntegrationTest.kt

```diff
@@ -598,4 +586,19 @@ class InteractiveShellIntegrationTest {
             waitForStateConsumption(stateRefs)
         }
     }
-}
\ No newline at end of file
+
+    @StartableByRPC
+    @InitiatingFlow
+    class FlowICanRun : FlowLogic<String>() {
```

Comment:
Why did you need to copy this func here from net.corda.tools.shell.SSHServerTest.FlowICanRun